packages/containers/src/List/List.container.js

```diff
@@ -216,6 +216,19 @@ class List extends React.Component {
 			}, {});
 		}
 
+		console.log(props.headerDictionary);
```

Comment:
please remove this console.log

Reply:
Ohhhhh i haven't seen it, this need to disappear 🔥